src/app/pages/courses-page/courses-page.component.ts

```diff
@@ -1,15 +1,18 @@
 import { Component, OnInit } from '@angular/core';
 
-import { Observable } from 'rxjs';
+import { Observable, of } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
 
 import { IconDefinition } from '@fortawesome/fontawesome-svg-core';
 import { faPlusCircle } from '@fortawesome/free-solid-svg-icons';
 
 import { Course } from '../../interfaces/course.interface';
+import { ConfirmService } from '../../services/confirm/confirm.service';
 import { CoursesService } from '../../services/courses/courses.service';
 import { LoadingService } from '../../services/loading/loading.service';
 
 const FIELD_NAME_FOR_COURSE_SEARCH = 'title';
+const DELETE_COURSE_CONFIRM_MESSAGE = 'Do you really want to delete this course?';
```

Comment:
Use can use an `enum` for different messages. This will allow you to use only one export/import and group different message types in one place. `export enum ErrorMessages/InfoMessages/... {}`

Reply:
Very useful advice, thanks